.github/workflows/ucentralgw-deployment.yaml

```diff
@@ -0,0 +1,125 @@
+name: Update uCentral
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  AWS_EKS_NAME: tip-wlan-main
+  AWS_DEFAULT_OUTPUT: json
+  AWS_DEFAULT_REGION: us-east-2
+  AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_CLIENT_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_CLIENT_KEY }}
+
+  # DigiCert certs
+  DIGICERT_CERT: ${{ secrets.DIGICERT_CERT }}
+  DIGICERT_KEY: ${{ secrets.DIGICERT_KEY }}
+
+  # https://stackoverflow.com/questions/59977364/github-actions-how-use-strategy-matrix-with-script
+  testbeds: '[
+    {
+      "namespace": "qa",
+      "ucentralgw_version": "master",
+      "ucentralgwui_version": "main"
+    }
+  ]'
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - feature/*
```

Comment:
Think this trigger should be removed, right?

Reply:
Yes, that's was introduced for test purposes. Deleted